src/app/app.component.ts

```diff
@@ -7,8 +8,13 @@ import { Component } from '@angular/core';
 })
 
 export class AppComponent {
-  logo: string = '../assets/logo.jpg';
 
+  constructor(private router: Router) {
```

Comment:
A good practice is to add readonly to injected services.
`constructor(private readonly router: Router) `

Reply:
done